emsdk/patches/0001-Don-t-set-toolchain-file-if-CMAKE_TOOLCHAIN_FILE-env.patch

```diff
@@ -0,0 +1,33 @@
+From a04edfb157cd8bd4f07abc9bafa9cb0e8c51e4d8 Mon Sep 17 00:00:00 2001
+From: Gyeongjae Choi <def6488@gmail.com>
+Date: Tue, 30 Aug 2022 05:57:51 +0000
+Subject: [PATCH] Don't set toolchain file if CMAKE_TOOLCHAIN_FILE env variable
```

Comment:
This patch is upstreamed and merged (https://github.com/emscripten-core/emscripten/pull/17757)